CONTRIBUTORS.yaml

```diff
@@ -1482,11 +1482,11 @@ tsenapathi:
     name: Tharindu Senapathi
     joined: 2019-05
 
-tklingstrom:
+TKlingstrom:
```

Comment:
shouldn't be necessary, github user names aren't case sensitive, but fine either way. 

if you're changing this, please also update topics/teaching/tutorials/experiences/2018-11-02-tklingstrom.md